hmp/utils.py

```diff
@@ -1467,6 +1486,20 @@ def condition_selection_epoch(epoch_data, condition_string, variable="event", me
 
 
 def participant_selection(hmp_data, participant):
+    """Select a participant from hmp_data.
+
+    Parameters
+    ----------
+    hmp_data : xr.Dataset
+        transformed EEG data for hmp, from utils.transform_data
+    participant : str | num
```

Comment:
This is not compatible with older versions of Python I think.

Reply:
Whate do you mean exactly?